pkgs/os-specific/linux/evdi/default.nix

```
@@ -13,13 +13,13 @@ let
 in
 stdenv.mkDerivation (finalAttrs: {
   pname = "evdi";
-  version = "1.14.1-unstable-2024-01-30";
+  version = "1.14.3";
 
   src = fetchFromGitHub {
     owner = "DisplayLink";
-    repo = "evdi";
-    rev = "d21a6ea3c69ba180457966a04b6545d321cf46ca";
-    hash = "sha256-Txa9yX9h3GfmHRRNvhrfrsUoQhqRWbBt4gJYAZTNe0w=";
+    repo = finalAttrs.pname;
```

```suggestion
    repo = "evdi";
```

using finalAttrs here will cause overrides that change `pname` to fail to build.